l10n_it_fatturapa_in_cassa_previdenziale/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Roberto Fichera <https://wwww.leveprime.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Italian Localization - Fattura elettronica - Integrazione '
```

Comment:
Come mai un modulo a parte?

Reply:
@eLBati Temporaneamente ho messo tutto in un modulo separato poi vediamo